src/super_gradients/training/metrics/segmentation_metrics.py

```diff
@@ -331,6 +331,13 @@ def update(self, preds, target: torch.Tensor):
         super().update(preds=preds, target=target)
 
 
+@register_metric("DIOU")
+class DIOU(IoU):
```

Comment:
What this class does, why is it here and why it has no docs?

Reply:
Accidentally left it there during development, removed now.